### PR TITLE
Separate noises

### PR DESCRIPTION
**Pull request by thuiop**
As we discussed yesterday, I separated the noise from the galaxy flux from the background noise ; it is possible to select which noise you want to be drawn.

### CODE REVIEW

File: btk/draw_blends.py [hunk 1]
```diff
@@ -417,12 +421,20 @@ def render_blend(self, blend_catalog, psf, filt, survey, seedseq_blend, extra_da
             _blend_image += single_image
 
         # add noise.
-        if self.add_noise:
+        if self.add_noise in ["galaxy", "all"]:
```
Comment (ismael-mendoza) [inline btk/draw_blends.py:424]: Doesn't matter too much but these lists you are creating only to check check conditions can be tuples.

**Review by ismael-mendoza — APPROVED** [review]
Looks good! Couldn't find anything wrong with the approach. I think you can merge whenever.

**Comment by ismael-mendoza** [comment]
@thuiop  I think you can merge this whenever, are there any parts of the documentation that will need updating besides the docstring?

**Comment by thuiop** [comment]
> @thuiop I think you can merge this whenever, are there any parts of the documentation that will need updating besides the docstring?
I'll look into it and merge afterwards.

**Comment by thuiop** [comment]
Ok, this is finally mergeable.

**Pull request merged**